src/frontend/next/src/theme/lightTheme.ts

```diff
@@ -24,7 +24,9 @@ export const lightTheme: Theme = createMuiTheme({
   },
 });
 
-export const darkTheme: Theme = createMuiTheme({
+export default lightTheme;
+
+/* export const darkTheme: Theme = createMuiTheme({
```

Comment:
Can you move this comment block to a `darkTheme.ts` file, so it exists, and we can file another issue to hook it up later?  Having it buried in a comment isn't great.